changelogs/client_server/newsfragments/1199.feature

```diff
@@ -0,0 +1 @@
+Add `room_types` filter and `room_type` response to `/publicRooms`, as per [MSC3827](https://github.com/matrix-org/matrix-spec-proposals/pull/3827).
```

Comment:
Should there be a newline here?

Reply:
The changelog system doesn't really care, so it depends on whether my editor decided to add it or not :p